src/ensembl/src/content/app/browser/BrowserCogList.tsx

```diff
@@ -55,6 +59,37 @@ const BrowserCogList = (props: BrowserCogListProps) => {
     return () => subscription.unsubscribe();
   }, []);
 
+  useEffect(() => {
+    if (props.browserCogTrackList) {
+      const ons: string[] = [];
+      const offs: string[] = [];
+
+      /* what the frontend and backend call labels and names is flipped */
+      Object.keys(props.browserCogTrackList).forEach((name) => {
+        /* undefined means not seen means on for names */
```

Comment:
This comment is a bit cryptic. Can this be rephrased?

Reply:
I completely agree, but that's not my comment; it came along with the code copied from BrowserImage. I will ask @ens-ds23 what he meant 😄

Reply:
@imran-ebi I have talked to Dan and updated the comments. I hope they make a bit more sense now.

Reply:
Yes that's much better.